src/ModernHttpClient/iOS/NSUrlSessionHandler.cs

```diff
@@ -46,7 +46,7 @@ public class NativeMessageHandler : HttpClientHandler
         public NativeMessageHandler(): this(false, false) { }
         public NativeMessageHandler(bool throwOnCaptiveNetwork, bool customSSLVerification)
         {
-            var config = NSUrlSessionConfiguration.EphemeralSessionConfiguration;
+            var config = NSUrlSessionConfiguration.DefaultSessionConfiguration;
```

Comment:
I think we probably also want to set URLCache to nil so that we prevent disk-based caching.